.mvn/wrapper/maven-wrapper.properties

```diff
@@ -16,4 +16,4 @@
 # under the License.
 wrapperVersion=3.3.2
 distributionType=only-script
-distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.9.8/apache-maven-3.9.8-bin.zip
+distributionUrl=https://archive.apache.org/dist/maven/mvnd/1.0.0/maven-mvnd-1.0.0-bin.zip
```

Comment:
as the next commit in this PR shows this is not a good idea

Reply:
If we can make it work and stable, why isn't that a good idea?

Reply:
It works. We can improve scripts to work with both mvnd and mvn. Is this ok with you?

Reply:
This would remove possibility to run the build with clean state.
I use ./mvnw if and only if i suspect mvnd may be messing up with the build.